jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -349,19 +349,23 @@ def test_population_count(self, harness: primitive_harness.Harness):
 
   @primitive_harness.parameterized(primitive_harness.lax_binary_elementwise)
   def test_binary_elementwise(self, harness):
-    if harness.params["dtype"] is dtypes.bfloat16:
-      raise unittest.SkipTest("bfloat16 not implemented")
-    if harness.params["lax_name"] in ("igamma", "igammac", "rem", "atan2"):
-      # b/158006398: TF kernels are missing for 'rem' and 'atan2'.
+    if harness.params["lax_name"] in ("rem", "atan2"):
```

Comment:
I think it makes sense at this point to save dtype and lax_name into variables before all these conditionals

Reply:
Solved!